extensions/wikia/LanguageWikisIndex/styles/language-wikis-index.scss

```diff
@@ -0,0 +1,72 @@
+@import 'extensions/wikia/DesignSystem/node_modules/design-system/dist/scss/wds-variables/index';
+
+.language-wiki-index {
+	margin: 50px auto 0;
+	max-width: 710px;
+	text-align: center;
+
+	&__intro {
+		font-size: 22px;
```

Comment:
Let's use variables from https://fandomdesignsystem.com/base-styles/typography. There is no 22px but IMO it only shows why we need to use standard variables. I'd make it `$wds-font-size-xl`